repose-aggregator/src/docs/asciidoc/recipes/getting-started.adoc

```diff
@@ -14,9 +25,124 @@ include::../_includes/in-progress.adoc[]
+However, these other methods do require a little more System Administration knowledge which is not covered here.
+See: <<Deployment>>
+
+==== Proxy Server (Valve)
+This is the most common setup and is detailed on the <<valve-installation.adoc#,Valve Installation>> page.
+
+After installing it this way, consult the <<running-valve.adoc#,Running Valve>> page for details on starting and stopping Repose.
+
+==== Servlet Container WAR
+Instructions for installing the WAR are on the <<war-installation.adoc#,WAR Installation>> page.
+
+===== Tomcat Setup
+
+After installing the WAR, there is additional setup to get everything working with Tomcat.
+See <<tomcat-setup.adoc#,Tomcat Setup>> for more details.
+
+===== GlassFish Setup
+
+See <<glassfish-setup.adoc#,GlassFish Setup>> for more details.
+
+===== Container Attributes known to affect Repose
+
+See <<http-container-attributes.adoc#,HTTP container attributes>>
+
+=== Deployment
+Once you know how you'll want to configure Repose, you may want to automate the deployment using a configuration management tool or reusable container.
+
+==== Vagrant
+If you don't already have a Linux environment ready, you can use Vagrant to easily create a new virtual machine running the Linux distribution of your choice.
+For more details, see: <<vagrant.adoc#,Vagrant>>
+
+==== Docker
+Docker and Repose make a fantastic team together.
+For more details, see: <<docker.adoc#,Docker>>
+
+==== Puppet
+There is an unsupported starter module in the GitHub repository https://github.com/rackerlabs/puppet-repose[rackerlabs/puppet-repose].
+
+==== Chef
+There is an unsupported starter cookbook in the GitHub repository https://github.com/rackerlabs/cookbook-repose[rackerlabs/cookbook-repose].
+
+=== Configuration
+If you're ready to dive into configuration, see <<../architecture/configuration.adoc#,Configuration>> for more details.
+We also have a series of <<index.adoc#,Recipes>> that will walk you through some of our common use cases.
+
+=== Performance Tuning
+Repose has been battle tested in production environments and in our performance testing environments.
+See <<performance-best-practices.adoc#,Performance Best Practices>> for more details.
+
+=== Support
+
+==== Recipes
+For further information on common configuration scenarios, visit our <<index.adoc#,Recipes>> page.
+
+==== Troubleshooting
+For details on common troubleshooting techniques, visit our <<../troubleshooting.adoc#,Troubleshooting>> page.
+
+==== FAQ
+For a list of frequently asked questions and answers, visit our <<../faq.adoc#,FAQ>> page.
+
+== Need More Information About Repose?
+<<../contact-us.adoc#,Contact us!>> We would be happy to address any questions, comments, or concerns with anything having to do with Repose!
 
-=== <<docker.adoc#,Docker>>
-Running Repose as a Docker container.
+TIP: If you wonder what we've been working on lately, visit our <<../release-notes.adoc#,release notes>>.
```

Comment:
The information about what Repose is shouldn't be here. It should probably live on the openrepose.org landing page.